docs/sql-ref-syntax-aux-conf-mgmt-reset.md

```diff
@@ -20,7 +20,7 @@ license: |
 ---
 
 ### Description
-Reset all the properties specific to the current session to their default values. After RESET command, executing SET command will output empty.
+Reset any runtime configurations specific to the current session which were set via the [SET](sql-ref-syntax-aux-conf-mgmt-set.html) command to their default values.
 
```

Comment:
We need to update the comment below, too?
```
-- Reset "all the properties" specific to the current session to their default values.
RESET;
```

Reply:
good catch!